package.json

```diff
@@ -67,6 +67,7 @@
     "@mdx-js/react": "^2.3.0",
     "autoprefixer": "^10.4.14",
     "cssnano": "^6.0.1",
+    "dotenv": "^16.4.7",
```

Comment:
Is this dependency necessary?

Reply:
Probably not if I just put the version aliases in a JS or JSON file from which all of nextjs config, code and tests can read. Shall I?

Reply:
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#loading-environment-variables

Reply:
I know but I use the .env in a test script as well.